objectRecognitionSource/README.md

```diff
@@ -389,19 +389,19 @@ No additional data is sent.
 
 ## Testing
 
-The test will download two large (~200 MB)files before running the test. If you do not want this to occur, do not run
+The test will download two large (~200 MB) files before running the test. If you do not want this to occur, do not run
 test or build.  
 
 There are some jni errors that will appear during testing that do not indicate test problems or failures. TensorFlow may
 display errors that look like `I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that
 this TensorFlow binary was not compiled to use:`. This merely indicates that there are improvements that could be made
 to performance by compiling the code specifically for your computer. If you wish to use these features, see [how to
 build TensorFlow Java](https://www.tensorflow.org/install/install_sources#build_the_c_or_java_libraries). Once built,
-either replace the `libtensorflow-<version>.jar` in `<install location>/objectRecognitionSource/lib` and run
-`<install location>/objectRecognitionSource/scripts/replaceJni <the built .so/.dll file>`; or add the locations of the
-new files into the `CLASSPATH` variable found in `<install location>/objectRecognitionSource/bin/objectRecognitionSource`
+set the environment variable `TENSORFLOW_JNI` to the folder containing all the .dll/.so/.dylib files, and either replace
+the `libtensorflow-<version>.jar` in `<install location>/objectRecognitionSource/lib` with the built jar or add the jar
+into the `CLASSPATH` variable found in `<install location>/objectRecognitionSource/bin/objectRecognitionSource`
 and `<install location>/objectRecognitionSource/bin/objectRecognitionSource.bat` and remove the `libtensorflow*` files
-in `<install location>/objectRecognitionSource/lib`.  
+in `<install location>/objectRecognitionSource/lib`. (This method is expected, but not certain not work).  
```

Comment:
Please remove the disclaimer.  If it doesn't work, they can file a bug.

You should be able to test this simply by moving the `libtensorflow-<version>.jar` file to somewhere else and making the same changes.  And/or renaming it (maybe).

Reply:
Works for moving the jni, and classpath mostly just depends on the users